wmfdata/spark.py

```diff
@@ -130,9 +214,10 @@ def run(commands, format="pandas", session_type="regular", extra_settings={}):
         * "pandas": a Pandas data frame
         * "raw": a list of tuples, as returned by the Spark SQL interface.
     * `session_type`: the type of Spark session to create.
-        * "regular": the default; able to use up to 15% of Hadoop cluster
+        * "local": Run the command in a local Spark process.
```

Comment:
Can you add the one sentence explanation of when to use a local session that you gave me? Also in the `get_session` docstring.

Reply:
Done.